glue/flagging_script_glue/flagging.py

```diff
@@ -728,7 +728,11 @@ def outlier_type(df: pd.DataFrame, condos: bool) -> pd.DataFrame:
             & (df["sv_pricing"].str.contains("High")),
             (df["sv_name_match"] != "No match")
             & (df["sv_pricing"].str.contains("High")),
-            (df["sv_transaction_type"] == "legal_entity-legal_entity")
+            (
+                df[["sv_buyer_category", "sv_seller_category"]]
+                .eq("legal_entity")
+                .any(axis=1)
+            )
```

Comment:
We changed this which allows for `non-person` flag to only require one `legal-entity` between the buyer and seller.

Reply:
**question:** To clarify, this will now generate a `non-person` flag when one side of the transaction is a legal entity AND the sale falls outside the `2,3` SD bounds (or there's another type of statistical flag)?

Reply:
That's correct, both conditions need to be satisfied.